string task_topic 
float32 task_timeout
bool nav_control
uint16 order
bool is_temp